packages/analytics/src/models/sessions.ts

```diff
@@ -0,0 +1,103 @@
+import { Command, Session } from '@linux-odyssey/models'
+import Pagination from './pagination.ts'
+import { Types } from 'mongoose'
+
+function formatTime(time: number) {
+  const seconds = Math.floor(time / 1000)
+  const minutes = Math.floor(seconds / 60)
+  const hours = Math.floor(minutes / 60)
+  return `${hours}h ${minutes % 60}m ${seconds % 60}s`
+}
+
+export async function sessionList(pagination: Pagination): Promise<any[]> {
```

Comment:
Prevent using any
```suggestion
import { ISession } from '@linux-odyssey/models'

export async function sessionList(pagination: Pagination): Promise<ISession[]> {
```
ref: [Mongoose TypeScript support](https://mongoosejs.com/docs/typescript.html)

Reply:
Oh, I'm wrong. It's a custom type actually. Create a interface or type for that.

Reply:
I created the interfaces under analytics/src/interface.ts